lib/hw_unique_key/hw_unique_key.c

```diff
@@ -82,6 +82,7 @@ bool hw_unique_key_are_any_written(void)
 	return false;
 }
 
+#ifdef CONFIG_HW_UNIQUE_KEY_USE
```

Comment:
This function is used by the TF-M here:
https://github.com/nrfconnect/sdk-nrf/blob/8d165848f383a74e262ee66e2bdfad35aca1cb20/modules/tfm/tfm/boards/common/crypto_keys.c#L39

So I am expecting the option CONFIG_HW_UNIQUE_KEY_USE  should probably be propagated to TF-M for when it needs to use this function?

Reply:
discussed offline